torch/autograd/grad_mode.py

```diff
@@ -59,3 +59,35 @@ def __enter__(self):
     def __exit__(self, *args):
         torch.set_grad_enabled(self.prev)
         return False
+
+
+class grad_mode(object):
+    r"""Context-manager that sets gradient calculation to on or off.
+
+    Parameter:
```

Comment:
Does `Parameter` work as a napoleon Docstring section? We've been using `Arguments:`

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/